tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Web/Startup.cs

```diff
@@ -429,7 +429,9 @@ private void CreateConfig(IHostingEnvironment hostingEnvironment)
                     jw.WriteValue(bool.Parse(generalSettingsSection["IsDevMode"]));
                     jw.WritePropertyName("isBackwardCompatible");
                     jw.WriteValue(bool.Parse(generalSettingsSection["IsBackwardCompatible"]));
-                jw.WriteEndObject();
+                    jw.WritePropertyName("isClientMappedWithHierachy");

```

Comment:
Added one more property in the general section of appsettings.json called "IsClientMappedWithHierachy". This will decide whether the taxonomy hierarchy needs to be filtered for client selectin or not